File: app/src/main/java/jerry/filebrowser/setting/SettingData.java
package jerry.filebrowser.setting;

import java.util.ArrayList;
import java.util.List;

public class SettingData {
    public int version;
    public ArrayList<String> colList;
    public ArrayList<Boolean> triggerList;
    public int option;

    public SettingData() {
        version = 1;
        colList = new ArrayList<>();
    }
}

File: app/src/main/java/jerry/filebrowser/setting/SettingData.kt
package jerry.filebrowser.setting

import java.util.ArrayList

class SettingData {
    var version = 1
    var option = 0
    var apiMode = 0

    @JvmField
    var colList: ArrayList<String> = ArrayList()

    @JvmField
    var triggerList: ArrayList<Boolean> = ArrayList()
}